src/utils/DeviceUtils.ts

```diff
@@ -53,9 +53,18 @@ export class DeviceUtils {
   }
 
   static isMobileDevice() {
+    if (this.isSafariIPadOS()) {
+      return true;
+    }
+
     return mobile;
   }
 
+  // Workaround for Safari on iPadOS https://developer.apple.com/forums/thread/119186
+  private static isSafariIPadOS(userAgent = navigator.userAgent) {
+    return userAgent.match(/Macintosh/i) && navigator.maxTouchPoints && navigator.maxTouchPoints > 2;
```

Comment:
It's unfortunate that this is necessary. If Macbooks have a touch screen in the future, I'm hoping iPadOS sends a different user agent.